src/Tools/BuildActionTelemetryTable/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using TelemetryInfo = System.Tuple<string, string, string>;
+
+namespace BuildActionTelemetryTable
+{
+    class Program
+    {
+        private static readonly string s_executingPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+
+        private static readonly string[] s_excludeList = new[]
+        {
+            "Microsoft.CodeAnalysis.AnalyzerUtilities.dll"
+        };
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Loading assemblies and finding CodeActions...");
+
+            var assemblies = GetAssemblies(args);
+            var codeActionTypes = GetCodeActionTypes(assemblies);
+
+            Console.WriteLine("Generating Kusto datatable of CodeAction hashes...");
+
+            var telemetryInfos = GetTelemetryInfos(codeActionTypes);
+            var datatable = GenerateKustoDatatable(telemetryInfos);
+
+            var filepath = Path.GetFullPath(".\\ActionTable.txt");
+
+            Console.WriteLine($"Writing datatable to {filepath}...");
+
+            File.WriteAllText(filepath, datatable);
```

Comment:
could be a future improvement, but maybe prompt to overwrite or just write to a new file and add int suffix

Reply:
Yeah. I don't think we will be running this too much. Might wait until we get a request.